source/interface/transport_interface.h

```diff
@@ -31,6 +31,12 @@
 #include <stdint.h>
 #include <stddef.h>
 
+/* *INDENT-OFF* */
+#ifdef __cplusplus
+    extern "C" {
+#endif
```

Comment:
Do we add this before or after other includes? I added it right after the `#define` header guard when I updated this file for coreMQTT

Reply:
FreeRTOS adds it after the other includes, and I guess that's because you wouldn't want to necessarily nest the other headers with an additional `extern "C" {`.